actions/send_email.py

```diff
@@ -5,7 +5,8 @@
 
 
 class SendEmail(Action):
-    def run(self, email_from, email_to, subject, message, account):
+    def run(self, email_from, email_to, subject, message, account, mime):
```

Comment:
It might be cleaner to give a default to the `mime` parameter and then ensure it value is in a list.

I.E.:
```
def run(self, email_from, email_to, subject, message, account, mime='plain'):
    if mime not in ['plain', 'html']:
        raise ValueError('Nope')
    ....
```

I'd also rather it raise here if the wrong value is provided.